src/components/Pages/Opportunities/Opportunities.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Project from '../../Project/Project'
+import TagCluster from '../../TagCluster/TagCluster'
+import { sortByTags, filterBySearch } from '../../../services/utils'
+import Search from '../../Search/Search'
+import Overlay from '../../Project/ProjectOverlay/Overlay'
+import Header from '../../Header/Header'
+
+export default function Opportunities({ onBack }) {
+  const [tags, setTags] = useState(null)
+  const [projects, setProjects] = useState(null)
+  const [searchTerm, setSearchTerm] = useState()
+  const [scrollPosition, setScrollPosition] = useState(0)
+
+  useEffect(() => {
+    fetch('https://unfinished-api.herokuapp.com/api/projects')
```

Comment:
Please use env variables and create services (single function files in a folder `services`) for that.